docs/discoveries/bb/hudanims/blb_hudanims.md

```diff
@@ -0,0 +1,17 @@
+---
+description: Info about Unused HUD Animations in Sonic Generations
+---
+There's some unused HUD animations for Time Eater in <code style="color:red;">bb.cpk/SonicActionCommonHud.ar.00\ui_gameplay.xncp</code>.
```

Comment:
This file is missing a title. Please specify a title using `#` in the markdown file.

Example:

```
---
description: Info about unused HUD animations in Sonic Generations
---

# Unused HUD Animations
There's some (...)
```

Reply:
Updated again